### PR TITLE
Show help for deprecated Kubelet flags

### PR DESCRIPTION
**Pull request by mtaufen**
We recently deprecated a bunch of Kubelet flags, which caused them to disappear from `--help` output. This PR unhides these flags, so that the deprecation notice is clearly visible in `--help`.

Fixes: #62009

```release-note
NONE
```

/cc @eparis

### CODE REVIEW

File: cmd/kubelet/app/options/options.go [hunk 1]
```diff
@@ -390,6 +401,8 @@ func (f *KubeletFlags) AddFlags(fs *pflag.FlagSet) {
 	fs.StringVar(&f.BootstrapCheckpointPath, "bootstrap-checkpoint-path", f.BootstrapCheckpointPath, "<Warning: Alpha feature> Path to to the directory where the checkpoints are stored")
 
 	// DEPRECATED FLAGS
+	fs.StringVar(&f.BootstrapKubeconfig, "experimental-bootstrap-kubeconfig", f.BootstrapKubeconfig, "deprecated: use --bootstrap-kubeconfig")
```
Comment (eparis) [inline cmd/kubelet/app/options/options.go]: This is going to result in some funny help text, something like:
```
--experminetal-bootstrap-kubeconfig string  deprecated: use --bootstrap-kubeconfig (DEPRECATED: Use --bootstrap-kubeconfig) (default: $f.BootstrapKubeconfig)
```

Comment (mtaufen) [reply cmd/kubelet/app/options/options.go]: updated

**Pull request merged**